.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "bracketSpacing": true,
+  "singleQuote": false,
+  "trailingComma": "es5",
+  "arrowParens": "always",
+  "jsxBracketSameLine": true,
+  "bracketSameLine": true,
+  "singleAttributePerLine": false,
+  "endOfLine": "lf",
+  "printWidth": 120
+}
```

Comment:
Make sure these properties match exactly

```json
{
    "tabWidth": 4,
    "semi": true,
    "singleQuote": false,
    "printWidth": 120,
    "jsxBracketSameLine": true
}
```

Also:
- `useTabs` should be true
- `singleAttributePerLine` should be true
- `endOfLine` should be removed, let prettier handle based on OS

Reply:
updated